src/github.com/getlantern/tarfs/tarfs.go

```diff
@@ -152,8 +153,7 @@ func (fs *FileSystem) get(p string, ignoreempty bool) ([]byte, error) {
 	}
 	b, found := fs.files[p]
 	if !found {
-		err := fmt.Errorf("%v not found", p)
-		return nil, err
+		return nil, os.ErrNotExist
```

Comment:
When I had a bad dist folder, I was missing files.  However, the web server reported a 500 error, which was confusing.  Returning os.ErrNotExist here causes it to behave normally and return a 404.